src/modules/back-to-top/back-to-top.component.ts

```diff
@@ -0,0 +1,35 @@
+import { Component, ElementRef, Input, OnInit, ViewChild } from '@angular/core';
+
+@Component({
+  selector: 'stache-back-to-top',
+  templateUrl: './back-to-top.component.html',
+  styleUrls: ['./back-to-top.component.scss']
+})
+export class StacheBackToTopComponent {
+  @Input() offset: number;
+  @Input() color: string;
+  @Input() backgroundColor: string;
+  @ViewChild('backToTop') el: ElementRef;
+
+  ngOnInit(): void {
+    window.addEventListener('scroll', () => {
+      if (window.pageYOffset > this.offset) {
+        this.el.nativeElement.classList.add('show');
```

Comment:
The partial support is for multiple parameters and specific types of DOM elements, neither of which I am using with this implementation. It should be fine. But agreed, we should test.